go-controller/pkg/libovsdbops/chassis.go

```diff
@@ -115,3 +137,122 @@ func DeleteChassisWithPredicate(sbClient libovsdbclient.Client, p chassisPredica
 	err := m.Delete(opModels...)
 	return err
 }
+
+func CreateOrUpdateRemoteChassis(sbClient libovsdbclient.Client, hostname, name, ip string) error {
```

Comment:
Ack.

Need to create the encapsulation record when creating the chassis, otherwise ovsdb-server will reject the transaction - https://github.com/ovn-org/ovn/blob/main/ovn-sb.ovsschema#L33